types/node/v16/test.d.ts

```diff
@@ -0,0 +1,176 @@
+/**
+ * The `node:test` module provides a standalone testing module.
+ * @see [source](https://github.com/nodejs/node/blob/v18.0.0/lib/test.js)
```

Comment:
this is obviously out-of-the-context, there should be `v16.*.*` in url, like e.g.:
https://github.com/nodejs/node/blob/v16.17.0/lib/test.js
(should be one feature was introduced)

Reply:
The same for:
```ts
@since v18.6.0
```